nntrainer/layers/embedding.cpp

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Jijoong Moon <jijoong.moon@samsung.com>
+ *
+ * @file	embedding.cpp
+ * @date	04 March 2021
+ * @brief	This is Embedding Layer Class of Neural Network
+ * @see		https://github.com/nnstreamer/nntrainer
+ * @author	Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug		No known bugs except for NYI items
+ *
+ */
+
+#include <embedding.h>
+#include <layer_internal.h>
+#include <lazy_tensor.h>
+#include <nntrainer_error.h>
+#include <nntrainer_log.h>
+#include <parse_util.h>
+#include <util_func.h>
+
+namespace nntrainer {
+
+const std::string EmbeddingLayer::type = "embedding_layer";
```

Comment:
how about setting type name as "embedding" to keep the convention consistent?

Reply:
done